source/mainnet/net/references/multi-sig.rst

```diff
@@ -0,0 +1,448 @@
+
+   where ``new-credential.json`` is the file from the previous step.
+
+   This command sends a transaction on-chain to add an additional credential to your account.
+   The ``<number-of-credential-to-sign>`` specifies the minimum number of credentials that are needed to sign the
+   transaction after the update.
+
+Configure the tool to use a multi-sig account
+=============================================
+
+To configure the ``concordium-client`` tool to use a multi-sig account, you can either:
+
+- pass in the signing keys via a file every time you sign a transaction (Option 1).
+- configure the ``concordium-client`` tool once to include the additional keys in its local key directory (Option 2).
+
+To view the key directory path that the ``concordium-client`` tool is using, run the following command:
+
+   .. code-block:: console
+
+      $concordium-client config show
+
+The goal is to add additional keys for your account to the local key directory of the ``concordium-client``
+tool in this section (Option 2). The guide for the first option is in the
+:ref:`following section<sign-with-keyfile>`.
+
+.. dropdown:: Import the keys by adapting the key export file format from the browser wallet
+
+   We recommend that you export a key file from a random account on the browser wallet and use this
+   JSON file as a template to generate a corresponding JSON file for the keys of the account you want to update.
+
+   .. note::
+
+      You can read up on how to export a key file from the browser wallet :ref:`here<export-key>`.
+
+   Your ``browser_wallet.export`` file should look similar to the following browser wallet export template:
+
+   .. code-block:: json
+      :force:
+
+      {
+         "type": "concordium-browser-wallet-account",
+         "v": 0,
+         "environment": "testnet",
+         "value": {
+            "accountKeys": {
+               "keys": {
+                  "0": {
+                     "keys": {
+                        "0": {
+                           "signKey": "<Key_0_Private_Key_Without_0x_Prefix>",
+                           "verifyKey": "<Key_0_Public_Key_Without_0x_Prefix>"
+                        },
+                        "1": {
+                           "signKey": "<Key_1_Private_Key_Without_0x_Prefix>",
+                           "verifyKey": "<Key_1_Public_Key_Without_0x_Prefix>"
+                        }
+                     },
+                     "threshold": 2
+                  }
+               },
+               "threshold": 1
+            },
+            "credentials": {
+               "0": "97f325c9f86066ab0c80ff879c21629eb67818841940869308d6a72886d18f8668e62e43ad228fdcbda245d0722454df"
+            },
+            "address": "4jxvYasaPncfmCFCLZCvuL5cZuvR5HAQezCHZH7ZA7AGsRYpix"
+         }
+      }
+
+   You can import the keys of this ``browser_wallet.export`` file and associate it to your account as follows:
```

Comment:
In the dropdown title here it states that you should "adapt" the browser wallet export. I don't see anything in the section regarding adapting the exported file?

Maybe a comment in the JSON file stating which section was added here, or some other way for users to easily identify what was added to the export file?

Reply:
While that is true, I think in some cases it could be useful to do anyway, as it would be easier for the user to identify exactly what is being added. Also, there actually is a JSON format with comments (`jsonc`), but that is probably not accepted by `concordium-client`.